clients/cobol-lsp-vscode-extension/src/services/snippetcompletion/cobolSnippets.json

```diff
@@ -188,6 +208,29 @@
     "body": ["BY ${1:item}"],
     "description": "BY Item"
   },
+  "INDEXED BY": {
+    "prefix": "BY",
+    "body": [
+      "01 ${1:TABLE-X}",
+      "\t05 ${2:RECORD-X} OCCURS ${3:100} TIMES INDEXED BY ${5:X-IND}.",
+      "\t\t10 ${6:elementary-items}"
+    ],
+    "description": "Defines an internal COBOL index for an internal COBOL table. https://publibfp.dhe.ibm.com/epubs/pdf/igy6lr40.pdf"
+  },
+  "SET x UP BY": {
+    "prefix": "BY",
+    "body": ["SET ${1:X-IND} UP BY ${2:1}"],
+    "description": "Adds +1 an internal COBOL table's index - assuming INDEXED BY. https://publibfp.dhe.ibm.com/epubs/pdf/igy6lr40.pdf"
+  },
+  "STRING x DELIMITED BY y": {
+    "prefix": "BY",
+    "body": [
+      "STRING ${1:VAR-1} DELIMITED BY ${2:VAR-2}",
+      "\tINTO ${3:VAR-3}",
+      "END-STRING."
+    ],
+    "description": "BY in the STRING (and UNSTRING) statements defines the transfer operation specifics: INTO VAR-3. https://publibfp.dhe.ibm.com/epubs/pdf/igy6lr40.pdf"
```

Comment:
Isn't there a better link to provide the details than a link to the PDF documentation?

Reply:
I'm not entirely convinced the links are necessary at all - if they were for attribution I'd understand but apparently it's just a "for more info" link, I think we would normally assume users can find that info themselves.
If you do keep them, linking to the online doc is probably better.